src/NuGet.Clients/PackageManagement.UI/PackageLoader.cs

```diff
@@ -43,9 +43,6 @@ internal class PackageLoader : ILoader
 
         private readonly string LogEntrySource = "NuGet Package Manager";
 
-        // The list of packages that have updates available
-        private List<UISearchMetadata> _packagesWithUpdates;
-
```

Comment:
This was used to improve performance by avoiding recalculation during paging. We now get all results in one go and there is no paging, so it's not needed.